test/e2e/generators/capi.go

```diff
@@ -49,7 +49,6 @@ func (g *ClusterAPI) releaseYAMLPath() string {
 
 // Manifests return the generated components and any error if there is one.
 func (g *ClusterAPI) Manifests(ctx context.Context) ([]byte, error) {
-	// TODO: this is not very nice
```

Comment:
@jadarsie is this still relevant? If so could you please file an issue with what you had in mind

Reply:
Not my comment actually, it was there in the first iteration of the test framework

https://github.com/kubernetes-sigs/cluster-api/commit/ea48d0cbce4443165cf035e062d2f95d9d6e918a#diff-c4cddf335f3f881e1b942982a3ada01f

Reply:
thanks, removed those files in #569 because I realized these were duplicates from the ones in the upstream framework.